File: SDKs/Aspose.Pdf_Cloud_SDK_for_Java/src/main/java/com/aspose/pdf/model/Images.java
package com.aspose.pdf.model;

import java.util.*;
public class Images {
  private List<Image> List = new ArrayList<Image>();
  private List<Link> Links = new ArrayList<Link>();
  /**
	 * getList
	 * Gets List<Image>
	 * @return List
	 */
  public List<Image> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<Image>
	 * @param List List<Image>
	 */
  public void setList(List<Image> List) {
    this.List = List;
  }

  /**
	 * getLinks
	 * Gets List<Link>
	 * @return Links
	 */
  public List<Link> getLinks() {
    return Links;
  }

	/**
	 * setLinks
	 * Sets List<Link>
	 * @param Links List<Link>
	 */
  public void setLinks(List<Link> Links) {
    this.Links = Links;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Images {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("  Links: ").append(Links).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf-Cloud-SDK-for-.NET/src/Com/Aspose/PDF/Model/Images.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class Images {
    public List<Image> List { get; set; }

    public List<Link> Links { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Images {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("  Links: ").Append(Links).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
